angular/base/src/app/app.component.spec.ts

```diff
@@ -26,9 +26,9 @@ describe('AppComponent', () => {
         { provide: Platform, useValue: platformSpy },
       ],
     }).compileComponents();
-  });
+  }));
 
-  it('should create the app', async () => {
```

Comment:
The test that this was on has no async calls. Analogous test in `ng cli` app _is_ wrapped in `async()` call even though it is not needed. However, if you generate a component via `ng g component foo` analogous test for that component is _not_ wrapped with `async()` so the ng cli is inconsistent with itself.

Choose to remove it here because the test immediately following this one _does_ require an `async/await` so having either a naked `async` or the `async()` function here for no reason just added confusion.

Reply:
Agreed